lms/djangoapps/teams/tests/test_views.py

```diff
@@ -529,9 +530,12 @@ def test_text_search(self, text_search, expected_team_names):
 
 
 @ddt.ddt
-class TestCreateTeamAPI(TeamAPITestCase):
+class TestCreateTeamAPI(EventTestMixin, TeamAPITestCase):
     """Test cases for the team creation endpoint."""
 
+    def setUp(self):  # pylint: disable=arguments-differ
```

Comment:
Would the `arguments-differ` warning go away if you were to add `*args, **kwargs` to the method signature?